fakecollector/collector.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# This is a fake collector that handles incoming HTTP POST requests to /submit and logs
+# headers, annotations, and files in the submission.
+
+import json
+import logging
+
+import click
+from werkzeug.exceptions import HTTPException, MethodNotAllowed
+from werkzeug.formparser import parse_form_data
+from werkzeug.routing import Map, Rule
+from werkzeug.wrappers import Request, Response
```

Comment:
The only web framework we have available is Django. I didn't want to use Django for the fake collector because that involves a whole bunch of additional set up for the db I didn't want to do.

Flask is an ergonomic shell on top of werkzeug. We don't need templates, localizations, extensions, etc so we can use raw werkzeug to build a very minimal API for `/submit`.